src/lib/kademlia/membership.ml

```diff
@@ -258,7 +258,8 @@ module Haskell_process = struct
                 line_no_prefix ;
               None
           | "DATA" ->
-              Logger.info logger ~module_:__MODULE__ ~location:__LOC__ "%s"
+              (* Too noisy to put in info logs *)
```

Comment:
Add some metadata that lets us distinguish this from `DBUG` above.

Reply:
ok